packages/react/popper/src/Popper.tsx

```diff
@@ -88,29 +143,29 @@ const Popper = React.forwardRef((props, forwardedRef) => {
 
   return (
     <div style={popperStyles} data-radix-popper-wrapper="">
```

Comment:
This is actually `data-radix-popper-content-wrapper` now but I'm worried about renaming it given consumers are using it... thoughts?

Reply:
I think we should just update it and document the breaking change. I sometimes worry about these things too but I don't think we should shy away from it. I'd rather we make the changes to keep the codebase sane.